test/lib/rules/grouped-require.js

```diff
@@ -11,10 +11,12 @@ test(RULE_NAME, async (t) => {
     valid: [
       {
         code: `
-          require('timers')
+          const dynamic = './something.js';
+          require('timers');
```

Comment:
Do we do this somewhere? this is generally considered a bad practice. I would almost want this to be a linting error

Reply:
It also kind of break with the check-require plugin as well I think

Reply:
yeah let's call this a bad idea